openpype/modules/deadline/abstract_submit_deadline.py

```diff
@@ -397,6 +397,15 @@ def update(self, data):
         for key, value in data.items():
             setattr(self, key, value)
 
+    def add_render_job_env_var(self):
+        """Check if in OP or AYON mode and use appropriate env var."""
+        if os.environ.get("USE_AYON_SERVER") == '1':
```

Comment:
```suggestion
        if AYON_SERVER_ENABLED:
```

Requires `from openpype import AYON_SERVER_ENABLED`